api/lang/fr/candidate_suspended_filter.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+use Illuminate\Support\Facades\Lang;
+
+return [
+    'active' => Lang::get('common.active', [], 'en'),
```

Comment:
Shouldn't these be falling back to 'fr'?

Reply:
Good catch! I accidentally rolled it into this commit :sweat: 

https://github.com/GCTC-NTGC/gc-digital-talent/pull/10701/commits/cd0f7f60fef51e493a7dd99455d905e856234927